CROSS_PROJECT_COUNCIL.md

```diff
@@ -0,0 +1,9 @@
+# Cross Project Council (CPC)
```

Comment:
I don't think we have consensus yet about this governance structure, this might be premature